android-agent/src/main/java/io/opentelemetry/android/internal/services/applifecycle/ApplicationStateListener.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.internal.services.applifecycle;
+
+/**
+ * Listener interface that is called whenever the instrumented application is brought to foreground
+ * from the background, or vice versa.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public interface ApplicationStateListener {
```

Comment:
This has been copied from [here](https://github.com/open-telemetry/opentelemetry-android/blob/5b80e00d41e83b6a1bf1d0c49667006dcc9fe600/instrumentation/common-api/src/main/java/io/opentelemetry/android/instrumentation/common/ApplicationStateListener.java#L15). It should have been just moved, but the original interface is still referenced in all the modules that haven't been migrated, so I'm planning to remove it from the original place after all the modules are migrated, otherwise we wouldn't get a green build in the PRs in the meantime.